doc/Caliptra.md

```diff
@@ -556,15 +556,6 @@ Refer to [Error Reporting and Handling](#error-reporting-and-handling) for detai
 
 ![](./images/BMD_Boot_flow.png)
 
-The Boot Media Dependent profile is less intrusive to integrations, but extends the TCB for Caliptra to include SoC ROM.  The verification of measurement mechanism integration includes:
```

Comment:
Same, can we make this commit disappear from the PR

Reply:
I'm a git novice and totally messed this up. I'm going to close this PR and start clean.